helm/athena/templates/deployment.yaml

```diff
@@ -40,6 +40,12 @@ spec:
       - name: {{ .Values.image.name }}
         image: "{{ .Values.registry.domain }}/{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
         imagePullPolicy: Always
+        env:
+        - name: ATHENA_KUBERNETES_CACERT
```

Comment:
Is the code reading the variable already?

Reply:
Yes. We configure viper to use env vars following this naming to override configuration here: https://github.com/giantswarm/athena/blob/509731f86c5c4d3ba9a3068f883ab1c3e9ff3e34/cmd/daemon/runner.go#L120-L124
I tested by removing the CA value from the athena configmap and could confirm that it was still part of the configuration. :)